chatmaild/src/chatmaild/dictproxy.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import os
+from socketserver import (
+    StreamRequestHandler,
+    ThreadingMixIn,
+    UnixStreamServer,
+)
+
+
+class DictProxy:
+    def __init__(self):
+        self.transactions = {}
+
+    def loop_forever(self, rfile, wfile):
```

Comment:
A bit wild, seeing `loop_forever` show up in all the tests. But it is obvious enough what it means I'd say :)